main/sdr.go

```diff
@@ -244,9 +236,12 @@ func (u *UAT) sdrConfig() (err error) {
 	} else {
 		log.Printf("\tResetBuffer Successful\n")
 	}
+
 	//---------- Get/Set Freq Correction ----------
 	freqCorr := u.dev.GetFreqCorrection()
 	log.Printf("\tGetFreqCorrection: %d\n", freqCorr)
+
+	u.ppm = getPPM(u.serial)
```

Comment:
added a print for serial at the top and ppm was already printed out for freq correction